<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="User-PC" name="AttemptAll2" tests="1" failures="1" timestamp="2 Dec 2016 12:51:38 GMT" time="0.004" errors="0">
  <testcase name="AttemptTest" time="0.002" classname="assignment_Student.AttemptSurey">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
	at Data.Read_Data.ReadData(Read_Data.java:29)
	at assignment_Student.AttemptSurey.AttemptTest(AttemptSurey.java:33)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
... Removed 10 stack frames]]>
    </failure>
  </testcase> <!-- AttemptTest -->
</testsuite> <!-- AttemptAll2 -->
